views/partials/header/header.hbs

```diff
@@ -1,4 +1,7 @@
     <header id="header" class="{{ addUpgradeClasses }}">
+      {{#if RECRUITMENT_BANNER_LINK }}
+      <div class="recruitment-banner"><a href="{{ RECRUITMENT_BANNER_LINK }}" target="_blank" rel="noopener noreferrer">{{ RECRUITMENT_BANNER_TEXT }}</a></div>
```

Comment:
No localization?

Reply:
Hmm ... if we have/had a generic recruitment text we could localize that. @tcinotto - should we have a default recruitment text string? Or should each recruitment link run have custom text?

Reply:
Either that, or maybe have custom strings and get them pre-translated in pontoon before publishing a survey. The `RECRUITMENT_BANNER_TEXT` could be used as a pontoon id.
Not sure if that'd be enough to get us localized banners, and fallback to en-US if a translation is missing.

I guess neither solution would work if we wanted to exclude some banners from specific locales (if we were promoting the VPN or something).

Reply:
Going with english-only recruitment right now.